src/redux/books/books.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
+import { v4 as uuidv4 } from 'uuid';
+const ADD_BOOK = "ADD_BOOK";
```

Comment:
- [ ] I kindly suggest that you follow the [ducks](https://github.com/erikras/ducks-modular-redux) pattern 

Wrong :x: 
`const ADD_BOOK = "ADD_BOOK";
`

Right :heavy_check_mark: 
`const ADD_BOOK = bookstore/books/ADD_BOOK`